python-threatexchange/threatexchange/cli/experimental_fetch.py

```diff
@@ -23,9 +25,15 @@ class ExperimentalFetchCommand(command_base.Command):
     """
 
     PROGRESS_PRINT_INTERVAL_SEC = 30
+    DEFAULT_REFETCH_SEC = 3600 * 24 * 85  # 85 days
 
     @classmethod
     def init_argparse(cls, ap) -> None:
+        ap.add_argument(
+            "--continuation",
```

Comment:
(not blocking): I think this should be the default behavior. The indicator types you care about should be in the collaboration config, and it should just always try fetching them all if called with no arguments.

Will accept as a #todo if you agree with that path

Reply:
Added todo